File: TodoApp.Business.Entities/ViewModels/RegisterUserViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TodoApp.Business.Entities.ViewModels
{
    public class RegisterUserViewModel
    {
        public RegisterUserViewModel()
        { 
            RememberMe = false;
        }
        [Required]
        public string UserName { get; set; }
        [Required]
        public string Password { get; set; }
        public string ConfirmPassword { get; set; }
        [Required]
        public string EmailAddress { get; set; }
        public bool RememberMe { get; set; } 
    }
}

File: TodoApp.Web.WebApplication/app/Model/RegisterUserViewModel.ts
export class RegisterUserViewModel {
    constructor() {
        this.RememberMe = false;
        this.EmailAddress = "";
        this.grant_type = "password";
    }
    public UserName: string;
    public Password: string;
    public ConfirmPassword: string;
    public EmailAddress: string;
    public RememberMe: boolean;
    public grant_type: string;
}